auth/options.go

```diff
@@ -10,14 +10,12 @@ import (
 type Options struct {
 	// ID is the services auth ID
 	ID string
-	// Secret is used to generate new tokens
+	// Secret is used to authenticate the service
 	Secret string
 	// Token is the services token used to authenticate itself
 	Token *Token
-	// Public key base64 encoded
+	// PublicKey for decoding JWTs
 	PublicKey string
-	// Private key base64 encoded
-	PrivateKey string
```

Comment:
Leave public/private keypair? This is relevant for Auth itself to use.

Reply:
Would we not want to set this in micro since go-micro has no use for it?

Reply:
I think we've nuked a lot of stuff here which is going to come back e.g jwt implementation that needs public/private keypair or any auth implementation for that matter that makes use of this. I would leave it.